modules/ROOT/pages/alternatives.adoc

```diff
@@ -1,37 +1,21 @@
 = Setting alternatives
 
-Due to an https://github.com/fedora-sysv/chkconfig/issues/9[ongoing issue] in how alternatives configurations are stored on the system, Fedora CoreOS systems can not use the usual `alternatives` commands to configure them.
+Now Fedora CoreOS systems can use the usual `alternatives` commands to configure the default command, as https://github.com/fedora-sysv/chkconfig/issues/9[issue] is fixed.
```

Comment:
Let's keep the previous content at the bottom as this still applies until everyone as moved to F41 based FCOS. Something like: "Starting with Fedora CoreOS based on Fedora 41, ...". The bottom section can be named "manual setup" or something like that.